src/components/data/SelectionDrawer.js

```diff
@@ -57,12 +57,8 @@ function SelectionToolbar(props) {
 
     // useEffect will run on toolbarRef value assignment
     useEffect(() => {
-        // The 'current' property contains info of the reference:
-        // align, title, ... , width, height, etc.
-        if (toolbarRef.current && getToolbarHeight) {
-            getToolbarHeight(toolbarRef.current.offsetHeight);
-        }
-    }, [toolbarRef, getToolbarHeight]);
+        setToolbarRef(toolbarRef);
+    }, [toolbarRef, setToolbarRef]);
```

Comment:
I think this can actually just be `setToolbarRef(toolbarRef);` without the `useEffect` because `useComponentHeight` is already tracking the ref with its own `useEffect`.  Small thing though

Reply:
It works without `useEffect` but throws a big stack trace in browser console the first time you open the Selection Drawer. So I will keep this.